browser/brave_rewards/rewards_tab_helper.h

```diff
@@ -11,7 +11,9 @@
 #include "base/macros.h"
 #include "brave/components/brave_rewards/browser/rewards_service_observer.h"
 #include "build/build_config.h"
+#if !defined(OS_ANDROID)
```

Comment:
same as above for all of these - headers should be grouped

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Fixed in 1105be6d38f16175dba23eef73a248f35c2819e3